README.md

```diff
@@ -1,10 +1,8 @@
 # Publishing API
 
-The Publishing API aims to provide _workflow as a service_ so that common
-publishing features can be written once and used by all publishing applications
-across Government. Content can be stored and retrieved using the API and
-workflow actions can be performed, such as creating a new draft or publishing an
-existing piece of content.
+The Publishing API aims to provide _workflow as a service_ so that common publishing features can be written once and used by all publishing applications across Government. Content can be stored and retrieved using the API and workflow actions can be performed, such as creating a new draft or publishing an existing piece of content.
```

Comment:
(We've deviated so much from this aim I don't think it's at all true, but definitely not something to fix in this PR!)